<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediCareWindow</class>
 <widget class="QWidget" name="MediCareWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indian Medi Care</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_Patient_Registration">
    <widget class="QLabel" name="label_Registration">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>-7</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Patient Registration Form</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Patient_Submit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>170</width>
       <height>150</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Examination">
        <property name="text">
         <string>Examination...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Inst">
        <property name="text">
         <string>Institutions...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Reports">
        <property name="text">
         <string>Reports...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>192</width>
       <height>198</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_patient_id">
         <property name="text">
          <string>Patient ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Patient_Name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dob">
         <property name="text">
          <string>DoB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_gender">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Hospital">
         <property name="text">
          <string>Hospital</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLCDNumber" name="lcdnumber_Patient_Id">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Patient_Name"/>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_Patient_dob"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_Gender"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_Inst_Name"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Examinations">
    <widget class="QLabel" name="label_Examinations">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>10</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Examinations</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>170</width>
       <height>150</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Patients">
        <property name="text">
         <string>Patients...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Inst_2">
        <property name="text">
         <string>Institutions...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Reports_2">
        <property name="text">
         <string>Reports...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>88</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Exam Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Exam Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Patient Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Weight (Kg)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Height (cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>167</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLCDNumber" name="lcdnumber_Exam_Id"/>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit_Exam_date">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_Exam_Patient_Name">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Weight"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Height"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_Exam_Desc">
        <property name="baseSize">
         <size>
          <width>120</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFormatting">
         <set>QTextEdit::AutoAll</set>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="undoRedoEnabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_Exam_submit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>83</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Inst_Details">
    <widget class="QLabel" name="label_Inst">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Institutions Details</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Inst_Submit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>83</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>170</width>
       <height>150</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Patients_2">
        <property name="text">
         <string>Patients...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Examination_2">
        <property name="text">
         <string>Exminations...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Reports_3">
        <property name="text">
         <string>Reports...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>76</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>151</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit_Inst_Name"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_Inst_Desc"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Reports">
    <widget class="QLabel" name="label_Reports">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>0</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Reports</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>170</width>
       <height>150</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Patient">
        <property name="text">
         <string>Patient...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Examination_3">
        <property name="text">
         <string>Examination...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdLinkButton_Inst_3">
        <property name="text">
         <string>Institutions...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>121</width>
       <height>251</height>
      </rect>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Institutions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView_Inst"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Patients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView_Patient"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Examinations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView_Exam"/>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>label_Reports</zorder>
    <zorder>listView_Inst</zorder>
    <zorder>listView_Patient</zorder>
    <zorder>listView_Exam</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>textBrowser</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
